tests/baselines/reference/noImplicitAnyNamelessParameter.errors.txt

```diff
@@ -0,0 +1,33 @@
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(2,17): error TS7051: Parameter has a name but no type. Did you mean 'x: string'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(2,25): error TS7051: Parameter has a name but no type. Did you mean 'x: C'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(3,19): error TS7051: Parameter has a name but no type. Did you mean 'x: C'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(3,22): error TS7051: Parameter has a name but no type. Did you mean 'x: number'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(4,20): error TS7051: Parameter has a name but no type. Did you mean 'x: boolean'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(4,29): error TS7051: Parameter has a name but no type. Did you mean 'x: C'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(4,32): error TS7051: Parameter has a name but no type. Did you mean 'x: object'?
+tests/cases/compiler/noImplicitAnyNamelessParameter.ts(4,40): error TS7051: Parameter has a name but no type. Did you mean 'x: undefined'?
+
+
+==== tests/cases/compiler/noImplicitAnyNamelessParameter.ts (8 errors) ====
+    class C { }
+    declare var x: (string, C) => void;
+                    ~~~~~~
+!!! error TS7051: Parameter has a name but no type. Did you mean 'x: string'?

```

Comment:
I decided to use `arg` instead of `param` because I think most people don't make this distinction and 'arg' is the more common term. (And I made the codefix behave the same way, except it also appends the index of the parameter to get `arg0, arg1, ...`.)

Reply:
Cool. Why not in error message also arg0, arg1?

Reply:
You're right, it's a good idea to show the exact code the codefix will generate.